src/figuresClasses.ts

```diff
@@ -1,11 +1,94 @@
-export interface Figure {}
+enum Color {
+  red = 'red',
+  green = 'green',
+  blue = 'blue',
```

Comment:
Enum constant names usually start with a capital letter. Consider renaming the enum values to start with a capital letter to follow this convention.